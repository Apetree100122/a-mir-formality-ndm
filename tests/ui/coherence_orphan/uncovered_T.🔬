[
    crate core {
        trait CoreTrait<ty T> where [] {}
    },
    crate foo {
        struct FooStruct<> where [] {}
        impl<ty T> CoreTrait<FooStruct<>> for T where [] {}
    }
]
