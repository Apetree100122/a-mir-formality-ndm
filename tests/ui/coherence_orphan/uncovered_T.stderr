Error: orphan_check(impl <ty> CoreTrait <FooStruct> for ^ty0_0 where [] { })

Caused by:
    failed to prove {@ IsLocal(CoreTrait(!ty_1, FooStruct))} given {}, got {}
